### DISCUSSION TITLE
Reset of Brick API key

### DISCUSSION THREAD

**Discussion by hometm** [BernhardSchlegel/BierBot-Bricks#4] [Q&A]
Pretty cool project!

I just set up the brick. Unfortunately, I pasted a wrong API key to my device. 

How can I change the API key after 'successful' network configuration?

Thanks, 
Tobias

**Comment by BernhardSchlegel** [comment]
Hi Tobias,

thanks for reaching out and the kind words.

Two options:
- Press button for 1-2s: Brick will start the configuration access point again.
- Hold button for 4s: Brick will reset all of it's settings (and will also start the configuration access point afterwards).

Did that work out for you?

Cheers,
Bernhard.

**Comment by shaktimaan** [comment]
Hallo Bernhard,

vielen Dank, daß Du dieses Projekt free of charge und open source veröffentlichst! 
Meinen tiefsten Respekt und Dank aus dem ÄppelWoi County!

Unfortunately the Reset doesn´t work for me in either way. 
Had to flash blank firmware from ESPEasy before i could reflash with Bier Brick

Thanks a lot - appreciate your work and afford,
Patrick
